Makefile

```diff
@@ -47,6 +47,7 @@ release:
       cd build.release;                          \
       export PATH=${BINPATH};                    \
       cmake -DCMAKE_BUILD_TYPE=RELEASE	       \
+  	  -DCMAKE_TOOLCHAIN_FILE="${CMAKE_TOOLCHAIN_FILE}"       \
```

Comment:
I've managed to unify the makefiles, such that the only difference is the inclusion of a CMAKE_TOOLCHAIN_FILE, which if not defined, then this argument is effectively ignored, so a regular non-cross-compiler build proceeds normally.